.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  push:
+    # Limit this action to tags only
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Yarn setup
+        uses: DerYeger/yarn-setup-action@master
+        with:
+          node-version: 16
+      - name: Release
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: |
+          touch .npmrc
+          echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> .npmrc
+          RELEASE_VERSION=${GITHUB_REF#refs/*/v}
+          npm --no-git-tag-version version $RELEASE_VERSION
+          npm publish
```

Comment:
"no new line at end of file"

Reply:
🤦 I saw this and then forgot. (fixed)

+1 for we need prettier